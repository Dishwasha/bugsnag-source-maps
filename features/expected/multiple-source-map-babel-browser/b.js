"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }

function _defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } }

function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); return Constructor; }

var B = /*#__PURE__*/function () {
  function B() {
    _classCallCheck(this, B);
  }

  _createClass(B, [{
    key: "hey",
    value: function hey() {
      console.log('hey');
    }
  }]);

  return B;
}();

exports["default"] = B;
//# sourceMappingURL=b.js.map
