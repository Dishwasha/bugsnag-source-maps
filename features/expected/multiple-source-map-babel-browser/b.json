{
    "version": 3,
    "sources": [
        "src/lib/b.js"
    ],
    "names": [
        "B",
        "console",
        "log"
    ],
    "mappings": ";;;;;;;;;;;;;IAAqBA,C;;;;;;;0BACb;AACJC,MAAAA,OAAO,CAACC,GAAR,CAAY,KAAZ;AACD",
    "sourcesContent": [
        "export default class B {\n  hey() {\n    console.log('hey')\n  }\n}\n"
    ],
    "file": "b.js"
}
